packages/frontend/src/components/RefreshButton.tsx

```diff
@@ -20,54 +21,37 @@ export const RefreshButton = memo(() => {
         (context) => context.hasUnfetchedChanges,
     );
 
+    const canRunQuery = !isLoading && isValidQuery && hasUnfetchedChanges;
+
     const { track } = useTracking();
 
     const onClick = useCallback(() => {
-        fetchResults();
-        track({ name: EventName.RUN_QUERY_BUTTON_CLICKED });
-    }, [fetchResults, track]);
-
-    const isButtonDisabled = isLoading || !isValidQuery || !hasUnfetchedChanges;
-
-    const hotkeys = useMemo<HotkeyConfig[]>(
-        () => [
-            {
-                combo: 'mod+enter',
-                group: 'Explorer',
-                label: 'Run query',
-                allowInInput: true,
-                onKeyDown: onClick,
-                global: true,
-                preventDefault: true,
-                stopPropagation: true,
-                disabled: isButtonDisabled,
-            },
-        ],
-        [onClick, isButtonDisabled],
-    );
+        if (canRunQuery) {
+            fetchResults();
+            track({ name: EventName.RUN_QUERY_BUTTON_CLICKED });
+        }
+    }, [fetchResults, track, canRunQuery]);
 
-    useHotkeys(hotkeys);
+    useHotkeys([['mod + enter', onClick, { preventDefault: true }]]);
 
     return (
         <Tooltip2
             content={
-                !hasUnfetchedChanges ? (
-                    'You need to make some changes before running a query'
-                ) : (
-                    <KeyCombo combo="mod+enter" />
-                )
+                !hasUnfetchedChanges
+                    ? 'You need to make some changes before running a query'
+                    : ''
```

Comment:
@PriPatel Mantine doesn't have a component to show key combos like blueprint had. Are we happy to remove them from our app our should we build our own component ?


<img width="179" alt="Captura de ecrã 2023-06-28, às 10 37 59" src="https://github.com/lightdash/lightdash/assets/9117144/e18b5e0c-0e87-4b23-b4b4-4dd46587ee75">

Reply:
@ZeRego  In that case, we should build our own as this is useful information to show (and for future shortcuts as well). Shall I open a ticket for this? I guess it will need design work first.

Reply:
@PriPatel found this mantine component https://mantine.dev/core/kbd/ that we can use. 

@muhsinkamil we could create a `HotKey` component that parses the string with `parseHotkey` and then render different symbols/keys depending on the user OS. 

```import { parseHotkey } from '@mantine/hooks/lib/use-hotkeys/parse-hotkey';```

We use similar logic as the blueprint component, how it maps the keys to icons/strings https://github.com/palantir/blueprint/blob/develop/packages/core/src/components/hotkeys/keyComboTag.tsx

Reply:
@muhsinkamil Maybe we should leave the current blueprint component for now and open a new ticket to migrate those components in a separate PR. What do you think ?

Reply:
> Maybe we should leave the current blueprint component for now and open a new ticket to migrate those components in a separate PR.

@ZeRego You mean a separate PR for parsing logic ? If so, I agree, I can try to make it in separate PR

Reply:
A separate PR to replace the HotKeys component. So we should leave the current one.